src/osx/cocoa/textctrl.mm

```diff
@@ -128,7 +128,7 @@ @interface wxTextEntryFormatter : NSFormatter
 
 @implementation wxTextEntryFormatter
 
-- (id)init 
+- (id)init
```

Comment:
It would be nice to avoid whitespace-only changes in this file.

Reply:
The white-space changes are in a separate commit. Is this really a problem?

Reply:
I didn't realize that they were in a separate commit, sorry. In this case it isn't a problem, although arguably it would still be better to avoid mixing significant and insignificant commits in the same PR (but it's not worth changing this).

More generally, concerning whitespace, I think we either shouldn't touch it at all or remove all trailing whitespace at once.